src/unix/linux_like/linux/mod.rs

```diff
@@ -2786,9 +2787,9 @@ pub const GETNCNT: c_int = 14;
 pub const GETZCNT: c_int = 15;
 pub const SETVAL: c_int = 16;
 pub const SETALL: c_int = 17;
-pub const SEM_STAT: c_int = 18;
+pub const SEM_STAT: c_int = if cfg!(musl32_time64) { 0x112 } else { 18 };
 pub const SEM_INFO: c_int = 19;
-pub const SEM_STAT_ANY: c_int = 20;
+pub const SEM_STAT_ANY: c_int = if cfg!(musl32_time64) { 0x114 } else { 20 };
```

Comment:
Ditto, may as well define them in terms of IPC_STAT https://github.com/bminor/musl/blob/1b76ff0767d01df72f692806ee5adee13c67ef88/include/sys/sem.h#L32-L34